backend/src/apiserver/storage/object_store.go

```diff
@@ -143,6 +144,27 @@ func (m *MinioObjectStore) GetSignedUrl(bucketConfig *objectstore.Config, secret
 		return "", err
 	}
 	reqParams := make(url.Values)
+	reqParams.Set("response-content-disposition", "attachment; filename=\""+key+"\"")
+	signedUrl, err := s3Client.Presign("GET", bucketConfig.BucketName, key, expirySeconds, reqParams)
+	if err != nil {
+		return "", util.Wrap(err, "Failed to generate signed url")
+	}
+
+	return signedUrl.String(), nil
+}
+
+func (m *MinioObjectStore) GetSignedUrlWithoutContentDisposition(bucketConfig *objectstore.Config, secret *v1.Secret, expirySeconds time.Duration, artifactURI string) (string, error) {
```

Comment:
What's the plan for test coverage of this method since the tests I see are mocking this out? Is this a separate task for an integration test?